### PR TITLE
df: always round up usage percentage (#3208)

### PR DESCRIPTION
**Pull request by cakebaker**
Always round up the usage percent. For example, `25.1%` is now rounded to `26%`, previously it was rounded to `25%`.

### CODE REVIEW

File: src/uu/df/src/table.rs [hunk 1]
```diff
@@ -482,4 +482,34 @@ mod tests {
             "my_device        my_type        4.0Ki        1.0Ki        3.0Ki   25% my_mount        "
         );
     }
+
+    #[test]
```
Comment (sylvestre) [inline src/uu/df/src/table.rs:486]: please also add a test in https://github.com/uutils/coreutils/blob/main/tests/by-util/test_df.rs

Comment (cakebaker) [reply src/uu/df/src/table.rs:544]: Ok, added in https://github.com/uutils/coreutils/pull/3209/commits/b167d18b48ddf4947b1fb8caa4781856b2b23bc3

**Comment by sylvestre** [comment]
Small test failure on Mac:
```
---- test_df::test_use_percentage stdout ----
current_directory_resolved: 
run: /Users/runner/work/coreutils/coreutils/target/x86_64-apple-darwin/debug/coreutils df
thread 'test_df::test_use_percentage' panicked at 'assertion failed: `(left == right)`
  left: `4`,
 right: `92`', /Users/runner/work/coreutils/coreutils/tests/by-util/test_df.rs:148:9


failures:
    test_df::test_use_percentage
```

**Comment by cakebaker** [comment]
I opened a new pull request that cleans up the mess with the commits.

**Pull request closed without merging**